backend/src/Response/Order/OrderCancelResponse.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Response\Order;
+
+class OrderCancelResponse
```

Comment:
We have update this class fields types in the previous pull request. Please, check why re-appeared here, and re-do the updates again.